srcs/main.cpp

```diff
@@ -30,13 +30,13 @@ int main(int argc, char **argv)
         std::cerr   << "Something went wrong when reading mesh file: "
                     << argv[1] << std::endl;
         return -1;
-    }
-    // displayMesh(mesh);
-    
-    // time-integrate the numerical scheme
-    if(!timeInteg(mesh, solverParams.timeIntType, solverParams.timeStep,
-                  solverParams.nbrTimeSteps, solverParams.solverType, 
-                  std::string(argv[1])))
+   }
+
+    omp_set_num_threads(2);
```

Comment:
You already tried with OpenMP ? Does it work with Travis too ?

Reply:
Well, it compiles and gives me results. It works with travis as you can see. Did not do speed test though.

Reply:
Ok ok, NIC4 is waiting for us then 😄  Everything is ok for me for the rest of the code. Can I merge ?

Reply:
You can ^^. I think I will stop for today.